package.json

```diff
@@ -20,8 +20,8 @@
   "author": "Nomic Labs SRL",
   "license": "MIT",
   "dependencies": {
-    "@resolver-engine/imports-fs": "^0.2.2",
-    "@solidity-parser/parser": "^0.6.0",
+    "@resolver-engine/imports-fs": "0.2.2",
+    "@solidity-parser/parser": "^0.7.0",
```

Comment:
Why not ^0.8.0?

Reply:
Because of I can't find a changelog on thier github repo for v0.8.0. Anyway just updated.

Reply:
Ohh, @fvictorio you may find this interesting

Reply:
Oh, yeah, sorry about that, I forgot to make that release. Here it is: https://github.com/solidity-parser/parser/releases/tag/v0.8.0